docs/parameters.md

```diff
@@ -0,0 +1,53 @@
+
+
+
+The user can provide the parameters below when configuring a storage class to dynamically provision a new FSx for OpenZFS **child volume**.
+
+Aside from ```volumeType```, each of the parameters below map to a corresponding parameter in the FSx API. So, **PLEASE read the documentation for the FSx [create-volume](https://docs.aws.amazon.com/cli/latest/reference/fsx/create-volume.html) API!**
+
+| Parameter                 | Required? | Sample Value                                                   | Notes                                                                                                |
+|---------------------------|-----------|----------------------------------------------------------------|------------------------------------------------------------------------------------------------------|
+| ```volumeType```          | Yes       | "volume"                                                       | Specifies whether the storage class will be used to create an FSx for OpenZFS file system or volume. |
+| ```parentVolumeId```      | Yes       | "fsvol-03062e7ff37662dff"                                      |                                                                                                      |
+| ```copyTagsToSnapshots``` | No        | "false"                                                        |                                                                                                      |
+| ```dataCompressionType``` | No        | "NONE"                                                         |                                                                                                      |
+| ```nfsExports```          | No        | "[{ClientConfigurations=[{Clients=*,Options=[rw,crossmnt]}]}]" |                                                                                                      |
+| ```readOnly```            | No        | "false"                                                        |                                                                                                      |
+| ```recordSizeKiB```       | No        | "128"                                                          |                                                                                                      |
+| ```userAndGroupQuotas```  | No        | "[{Id=1,StorageCapacityQuotaGiB=10,Type=USER}]"                |                                                                                                      |
+| ```tags```                | No        | "Tag1=Value1,Tag2=Value2"                                      |                                                                                                      |
+
+## KEY NOTES:
```

Comment:
I feel like this should be a document of its own, whether a design doc or a FAQ. I can think of a ton of things we should point out to a user.

Quick list off the top of my head:
* An incorrect parameter will not cause the driver to error. Therefore misspelling a parameter may result in unexpected behavior.
* Incorrectly formatted objects such as nfsExports will cause errors
* Values must exactly match API enums

Reply:
Agreed, I think we should make a separate FAQ with all of these considerations. One of us should create an FAQ listing all of these considerations. Since things are still changing, I'm partial to maintaining an internal document for this and publishing it to this repo when we're ready to launch.